apps/vmq_server/test/vmq_hook_rewrite_SUITE.erl

```diff
@@ -217,14 +217,14 @@ hook_on_publish_modified_payload(_UserName, _SubscriberId, _QoS, _Topic, <<"rewr
 hook_on_publish_modified_payload(_UserName, _SubscriberId, _QoS, _Topic, Payload, _IsRetain) ->
     throw({expected_payload, <<"rewritten">>, got, Payload}).
 
-hook_on_deliver(_User, {"", <<"dlvr-rewrite-test">>}, [<<"dlvr">>, <<"rewrite">>, <<"payload">>],
-                <<"message">>) ->
+hook_on_deliver(_User, {"", <<"dlvr-rewrite-test">>}, _, [<<"dlvr">>, <<"rewrite">>, <<"payload">>],
```

Comment:
@mths1 We don't necessarily have to change the function arity here. (it's a module hook). Did you see errors from this?

Reply:
Yep, either vmq_plugin_mgr or the test case starts complaining...